<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetWord</class>
 <widget class="QDialog" name="SetWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set word</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="word_for_correctLine">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="new_wordLine">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setButton">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="mode_comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
